cookbooks/aws-parallelcluster-computefleet/recipes/install/node.rb

```diff
@@ -35,27 +35,7 @@
 
 if !node['cluster']['custom_node_package'].nil? && !node['cluster']['custom_node_package'].empty?
   # Install custom aws-parallelcluster-node package
-  bash "install aws-parallelcluster-node" do
-    cwd Chef::Config[:file_cache_path]
-    code <<-NODE
-      set -e
-      [[ ":$PATH:" != *":/usr/local/bin:"* ]] && PATH="/usr/local/bin:${PATH}"
-      echo "PATH is $PATH"
-      source #{virtualenv_path}/bin/activate
-      pip uninstall --yes aws-parallelcluster-node
-      if [[ "#{node['cluster']['custom_node_package']}" =~ ^s3:// ]]; then
-        custom_package_url=$(#{cookbook_virtualenv_path}/bin/aws s3 presign #{node['cluster']['custom_node_package']} --region #{node['cluster']['region']})
-      else
-        custom_package_url=#{node['cluster']['custom_node_package']}
-      fi
```

Comment:
[Nitpick] This could be moved into an helper function and then we can store `node_package_url`  as part of the pcluster config. 
This will allow us also to test if we have a custom node or not.

Reply:
That's inside a bash block for the time being. I agree that we should rewrite that recipe in a more chef-friendly way, but I would keep it out of scope for now (it can be part of our cookbook refactoring later on).